src/components/HorizontalTab/ChannelListTabItem.tsx

```diff
@@ -9,7 +9,8 @@ export interface ChannelListTabItemProps {
   picture: string | null;
   name: string;
   unreadCount: number;
-  testID: string;
+  type: 'SIGNED' | 'ANONYMOUS';
+  testID?: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `type` property has been added to the component's props and its type is defined as either 'SIGNED' or 'ANONYMOUS'. This change seems to be in line with the PR title, which mentions a signed chat feature. However, the `testID` property is now optional (`testID?`). If this property is used for testing purposes and it's not guaranteed to be present, make sure your tests handle the case when `testID` is undefined.
```diff
-  testID: string;
+  testID?: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->